app/views/miq_ae_customization/_dialog_field_form_non_dynamic_options.html.haml

```diff
@@ -0,0 +1,253 @@
+- url = url_for(:action => 'dialog_form_field_changed', :id => @record.id.to_s || 'new')
+- if @edit[:field_typ] =~ /Text|Check/
+  .form-group
+    %label.col-md-2.control-label
+      = _('Default Value')
+    .col-md-8
+      - if @edit[:field_typ] =~ /TextArea/
+        = text_area_tag("field_default_value",
+          @edit[:field_default_value],
+          "data-miq_observe" => {:interval => '.5', :url => url}.to_json,
+          :maxlength         => 8196,
+          :size              => "50x6")
+      - elsif @edit[:field_typ] =~ /TextBox/
+        - if @edit[:field_protected]
+          = password_field_tag('field_default_value__protected',
+                                @edit[:field_default_value],
+                                'data-miq_observe' => {:interval => '.5', :url => url}.to_json)
+        - else
+          = text_field_tag("field_default_value",
+                            @edit[:field_default_value],
+                            "data-miq_observe" => {:interval => '.5', :url => url}.to_json,
+                            :class => "form-control",
+                            :maxlength         => 50)
+      - else
+        = check_box_tag("field_default_value", "1",
+                        @edit[:field_default_value],
+                        "data-miq_observe_checkbox" => {:url => url}.to_json)
+  - if @edit[:field_typ].include?('TextBox')
+    .form-group
+      %label.col-md-2.control-label
+        = _('Value Type')
+      .col-md-8
+        = select_tag('field_data_typ',
+                      options_for_select([[_("Integer"), "integer"], [_("String"), "string"]],
+                                          @edit[:field_data_typ]),
+                      "data-miq_sparkle_on" => true,
+                      :class    => "selectpicker")
+        :javascript
+          miqSelectPickerEvent('field_data_typ', "#{url}")
+    .form-group
+      %label.col-md-2.control-label
+        = _('Protected')
+      .col-md-8
+        = check_box_tag('field_protected', 'true',
+                        @edit[:field_protected],
+                        "data-miq_observe_checkbox" => {:url => url}.to_json)
+  .form-group
+    %label.col-md-2.control-label= _('Required')
+    .col-md-8
+      = check_box_tag('field_required', 'true',
+                      @edit[:field_required],
+                      "data-miq_observe_checkbox" => {:url => url}.to_json)
+  .form-group
+    %label.col-md-2.control-label= _('Read only')
+    .col-md-8
+      = check_box_tag('field_read_only', '1', @edit[:field_read_only],
+                      "data-miq_observe_checkbox" => {:url => url}.to_json)
```

Comment:
Similar to the 'Auto refresh other fields when modified' form group, I think this read_only field can be DRY'd up, but only within this file since the dynamic dialog fields don't use it. I guess technically it won't be in the same order as before, but I don't think that really matters.